pkg/database/realm_test.go

```diff
@@ -501,6 +501,59 @@ func TestRealm_UserStats(t *testing.T) {
 	}
 }
 
+func TestRealm_FindRealm(t *testing.T) {
+	t.Parallel()
+
+	db, _ := testDatabaseInstance.NewDatabase(t, nil)
+
+	realm1 := NewRealmWithDefaults("realm1")
+	realm1.RegionCode = "US-MOO"
+	if err := db.SaveRealm(realm1, SystemTest); err != nil {
+		t.Fatal(err)
+	}
+
+	realm2 := NewRealmWithDefaults("realm2")
+	if err := db.SaveRealm(realm2, SystemTest); err != nil {
+		t.Fatal(err)
+	}
+
+	// FindRealm by ID
+	{
```

Comment:
Instead of a scope, can you try using a sub-test?

```go
t.Run("by_id", func(t *testing.T) {
  t.Parallel()

  // ...
})
```

That will allow them to run in parallel (faster) and keep a clean scope.

Reply:
made it a table test